src/ContainerCommitModal.jsx

```diff
@@ -91,7 +91,8 @@ class ContainerCommitModal extends React.Component {
         }
         commitData.changes.push(...onbuildsArr);
 
-        varlink.call(utils.PODMAN_ADDRESS, "io.podman.Commit", commitData)
+        const more = true;
```

Comment:
Why does this need to be a constant? Just to avoid a bare "true" argument and make it more legible? Or is there some hidden magic that I don't understand?

Reply:
Just to avoid bare 'true' that noone has any idea what is represents.